### ISSUE TITLE
UVE: Losing content when changing languages

### ISSUE THREAD

**Issue by zJaaal** [dotCMS/core#30878]
### Problem Statement

We are losing content when we create a translation of page and add content to it. If you go to the original version of the page, you can see that some content is missing from the page. 

### Steps to Reproduce

https://github.com/user-attachments/assets/b3508eb9-4ef8-4394-a18f-e274f4b25fba

### Acceptance Criteria

- We should be able to create translations without messing with the original content

### dotCMS Version

trunk-latest

### Proposed Objective

User Experience

### Proposed Priority

Priority 1 - Show Stopper

### Assumptions & Initiation Needs

- You understand how translations work on UVE and dotCMS

**Comment by zJaaal** [comment]
I couldn't find any root causes in the FE. @fabrizzio-dotCMS Is helping me investigating the BE. He found this issue while testing https://github.com/dotCMS/core/issues/30897

**Comment by wezell** [comment]
Fabro!   IQA beast.

**Comment by zJaaal** [comment]
I worked along side @fabrizzio-dotCMS and @jgambarios and only Fabrizzio and I could reproduce it. 

We got to a point that it looked like only the Headless version of UVE was presenting this issue so I started to dig in again in the Frontend.

For some reason since I started my investigation today in the FE after several attempts testing the original STRs in different conditions and configurations I couldn't reproduce it again 

I cleaned my environment and set it up again and I couldn't reproduce it either. I'm not sure what changed since I didn't made anything weird in my env apart from the same steps I do to clean and set it up on normal basis, so I can not tell what is the root cause or what fixed it if it is fix.

I will keep an eye on this anyways, since it is blocker and it could be really annoying to a customer when building pages.

 cc: @nollymar @fmontes @wezell
